andela/metis-ah-frontend
# #15998735 setup testing with jest and enzyme

**What does this PR do?**
Setup testing with jest and enzyme

**Description of Task to be completed?**

- install `jest`, `enzyme`, `react`, `enzyme-adapter-react-16`
- setup `tests` folder
- add test scripts
- add a basic test suite

**How should this be manually tested?**

Clone the **repository**

Check out the `ch-setup-testing-159987356` branch
Run `npm install` on your local machine to install required Dependencies

Run `npm test`

**What are the relevant pivotal tracker stories?**
#159987356
